noxfile.py

```diff
@@ -15,4 +15,8 @@ def docs(session):
 def tests(session):
     session.install('invoke', './[test]')
     session.run('invoke', 'pytest', '--junit', '--no-pty')
+
+    session.chdir('./plugins/ext_test')
+    session.install('.[test]')
+    session.run('invoke', 'pytest', '--junit', '--no-pty', '--append-cov')
     session.run('codecov')
```

Comment:
Something is wrong with `codecov` in this PR - it isn't reporting data.  I think you need to move `session.run('codecov')` back above the `chdir` command.

Reply:
Oh, I think I need to chdir back to the root.

Reply:
I don’t think we want to measure code coverage of plugins or at least we don’t want to report that as part of the cmd2 coverage.

Reply:
I don't see why that's a problem. These plugins are optionally installed first-class components maintained by cmd2.

Reply:
I guess it is fine as long as it is clear exactly what coverage is being measured

Reply:
This seems to be fixed